HotelReservation/src/main/java/Request.java

```diff
@@ -0,0 +1,31 @@
+import com.sun.corba.se.impl.encoding.TypeCodeOutputStream;
+
+public class Request {
```

Comment:
You have added lambok dependency in pom but you are not using that. By using that you don't need to write getter setter equal and hashcode function by your own